src/Symfony/Component/Serializer/Tests/Encoder/YamlEncoderTest.php

```diff
@@ -54,14 +54,20 @@ public function testSupportsDecoding()
         $this->assertFalse($encoder->supportsDecoding('json'));
     }
 
+    /**
+     * The legacy group must be reverted on 3.4 and the php/object tag without a following colon must be used instead.
+     * @group legacy
```

Comment:
Maybe `"!php/object 'O:8:\"stdClass\":0:{}'" === Yaml::dump(new \stdClass())`? Not sure that's worth it though.